mindmeld/models/text_models.py

```diff
@@ -45,13 +46,13 @@
 DECISION_TREE_TYPE = "dtree"
 RANDOM_FOREST_TYPE = "rforest"
 SVM_TYPE = "svm"
+MLP_TYPE = "mlp"
 SUPER_LEARNER_TYPE = "super-learner"
```

Comment:
The rest of the super-learner was stripped out. We should remove this line too.

Reply:
Removed